### PR TITLE
Fix indices docstrings, DataFrameGroupBy, SeriesGroupBy, Resampler

### PR DESCRIPTION
**Pull request by gvmmybear**
- [x] Part of issue #60365
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

### CODE REVIEW

File: pandas/core/groupby/groupby.py [hunk 1]
```diff
@@ -515,6 +515,15 @@ def indices(self) -> dict[Hashable, npt.NDArray[np.intp]]:
         """
         Dict {group name -> group indices}.
 
+        See Also
+        --------
+        core.groupby.DataFrameGroupBy.indices : Provides a mapping of group names to
+         positions of the elements in object.
```
Comment (rhshadrach) [inline pandas/core/groupby/groupby.py:521]: Indent continuations by 4 spaces rather than one.

Comment (rhshadrach) [reply pandas/core/groupby/groupby.py:521]: `group rows` instead of `elements` I think. And it seems to me you can remove `in object`.

**Review by rhshadrach — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: pandas/core/groupby/groupby.py (hunk 1 above)
Comment (gvmmybear) [reply pandas/core/groupby/groupby.py]: Thank you for reviewing!

**Review by rhshadrach — APPROVED** [review]
lgtm

**Comment by rhshadrach** [comment]
Thanks @gvmmybear!

**Pull request merged**